### DISCUSSION TITLE
Xiaomi Map card - Select default map

### DISCUSSION THREAD

**Discussion by CarnifexRex** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#444] [Q&A] [answered]
On the xiaomi vacuum map card, how do you specifiy which map is displayed by default?  Rather than use some of the solutions I've seen for implementing map switching within one card, I want to make two cards, each designed to work for 1 floor of the house.  I have tried the map switching and been able to get that to work, but would rather have 2 cards. However the stock card only shows my downstairs, how can I get a card to display the upstairs map?

**Comment by PiotrMachowski** [comment, marked as answer]
Live mode shows only current active map (it's a current limitation of map extractor). You can use a map image instead to make it show only selected floor

**Reply by CarnifexRex** [reply]
Thanks. I didn't think of just loading an image, I can do that, Although I'll miss the interaction with the live map

**Comment by cataseven** [comment]
> On the xiaomi vacuum map card, how do you specifiy which map is displayed by default? Rather than use some of the solutions I've seen for implementing map switching within one card, I want to make two cards, each designed to work for 1 floor of the house. I have tried the map switching and been able to get that to work, but would rather have 2 cards. However the stock card only shows my downstairs, how can I get a card to display the upstairs map?

How can you manage to switch map in card?

**Reply by PiotrMachowski** [reply]
Using `additional_presets`, like [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/blob/master/docs/demo_config.yaml#L92)
